util/tls/tls_socket.h

```diff
@@ -148,20 +148,18 @@ class TlsSocket final : public FiberSocketBase {
     SHUTDOWN_IN_PROGRESS = 4,
     SHUTDOWN_DONE = 8,
   };
-  uint8_t state_{0};
+  uint8_t state_{};
```

Comment:
why this is correct, I do not understand the reason of changing the perfectly good code based on subjective aesthetic preferences.